src/pages/ProfilesExplorerView/components/SceneLabelValuesBarGauge.tsx

```diff
@@ -53,7 +56,11 @@ export class SceneLabelValuesBarGauge extends SceneObjectBase<SceneLabelValuesBa
         return;
       }
 
-      body.setState(this.getConfig(item, state.data.series));
+      const { series } = state.data;
+
+      this.publishEvent(new EventDataReceived({ series }), true);
```

Comment:
Publishing an event that can be listened to. Helps decoupling the code responsible for removing the panels without data in `SceneByVariableRepeaterGrid`.